instruqt/airgap-embedded/vendor/manifests/kots-config.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: kots.io/v1beta1
+kind: Config
```

Comment:
Is this needed? For this track, it doesn't look like we're doing anything with the config

Reply:
I wan to show the config screen, and I do have configuration there to set.